man/nvim.1

```diff
@@ -336,8 +336,22 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev XDG_CONFIG_HOME
+Path to use for the user-local runtime directory, see
+.Sx FILES .
+Defaults to
+.Pa ~/.config
```

Comment:
It's not really our place to mention a default for `$XDG_CONFIG_HOME`--if I'm not mistaken that may vary between distros. Even if it doesn't vary per distro, it may be confusing to mention it in the Neovim manpage. What's relevant to Neovim is:

- if user (or OS) set `$XDG_CONFIG_HOME` then we respect it 
- else we look for `~/.config/nvim`